packages/react-aria-components/src/Collection.tsx

```diff
@@ -741,7 +741,7 @@ export interface ItemRenderProps {
   isPressed: boolean,
   /**
    * Whether the item is currently selected.
-   * @selector [aria-selected=true]
+   * @selector [data-selected=true]
```

Comment:
I think we can do just `[data-selected]` without the `=true` here since we only add the attribute if it is true. That would make it more consistent with our other data attributes.